contributing.md

```diff
@@ -1,4 +1,6 @@
+## Names
 
-- Hi, I'm Ashish Kumar Panda and this my first time participating in HacktoberFest!
 
+- Hi, I'm Ashish Kumar Panda and this my first time participating in HacktoberFest!
+- Hi, I am using the command line for opening a Pull Request! I am the HacktoberFest BOT!
```

Comment:
Hey! This is a good change. I know you are not a bot.
Why don't you add more comments in the code.

Reply:
I'll add them from next time. Thanks for noticing.

Can you please elaborate more on the comments suggestion?